shapes/contacts/person.shex

```diff
@@ -0,0 +1,47 @@
+PREFIX vcard: <http://www.w3.org/2006/vcard/ns#>
+PREFIX xsd:   <http://www.w3.org/2001/XMLSchema#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX dc: <https://dublincore.org/specifications/dublin-core/dcmi-terms/#>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+
+# We should store the preferred WebId for each person, since sameAs can be multiple WebIds
+
+vcard:Individual {
+    vcard:fn xsd:string {1} ; 
+    vcard:inAddressBook vcard:AddressBook + ;
+    owl:sameAs vcard:Individual * ; # WebIds
+    vcard:hasUID xsd:string ? ;
+    vcard:hasName xsd:string ? ; # structured name 
+    vcard:hasPhoto dc:Image * ; 
+    vcard:hasRelated vcard:RelatedType * ; 
+    vcard:url rdf:Resource * ; # need to describe the type of url to be able to identify WebIds 
+    vcard:hasAddress vcard:Address * ; 
+    vcard:bday xsd:date ? ; 
+    vcard:hasEmail vcard:Email * ; 
+    vcard:hasTelephone vcard:Tel * ; 
+    vcard:organization-name xsd:string ? ; 
+    vcard:role xsd:string ? ; 
+    vcard:title xsd:string ? ; 
+    vcard:note xsd:string ? ;
+}
+
+vcard:Address {
+    vcard:street-address xsd:string {1} ;
+    vcard:locality xsd:string {1} ; 
+    vcard:postal-code xsd:string {1} ;
+    vcard:region xsd:string {1} ;
+    vcard:country-name xsd:string {1} ;
+}
+
+vcard:Email {
```

Comment:
While there is nothing wrong with the Email description, it is very limited only having an email address. It would be nice to have at least the type of email. Plus it would be good to have the regex to validate the email format. The https://solidsdk.inrupt.net/sdk/userprofile.shex describes an email as:
```
  <#UserProfileEmail> {
    a [vcard:Dom vcard:Home vcard:ISDN vcard:Internet vcard:Intl vcard:Label vcard:Parcel vcard:Postal vcard:Pref vcard:Work vcard:X400] {1}
      // rdfs:label "Type"@en
      // rdfs:label "Tipo"@es ;
    vcard:value xsd:string /^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,}$/ {1}
      // rdfs:label "Email"@en 
      // rdfs:label "Correo"@es
      // :prefix "mailto:";
  }
```
A few comments:

- Do not have to include `rdfs:label`s, although good for translations.
- ShEx does not have the concept of dynamically populated lists, hence the hard-coded list values.

Reply:
> [vcard:Dom vcard:Home vcard:ISDN vcard:Internet vcard:Intl vcard:Label vcard:Parcel vcard:Postal vcard:Pref vcard:Work vcard:X400]

Definitely makes sense to have email type. One question: do we need all those types? Other than Home and Work, they all seem to have been deprecated.

Reply:
@VirginiaBalseiro I think we went through the Regexes with @timbl ?

Reply:
@megoth I don't remember going over the regexes, please feel free to remove them if you think we shouldn't have them there.

Reply:
Ok, I probably remember wrong then.

@timbl Do you think we should use the Regexes suggested in the ShEx-shapes, or those in the SHACL shape, or something different?

Reply:
Re: Email Types. A smaller selection of email (and phone) tyoes would be fine. The list shown are the ones used in Data Browser. If the types could be dynamically generated (and in the User Profile Form Model used by React SDK v1 Generator https://solidsdk.inrupt.net/sdk/userprofile.ttl), the list of types would be:
- Email: Home, Work
- Phone: Cell, Fax, Pager, Text, TextPhone, Video, Voice